docs/templates/reachview/firmware-reflashing.mdx

```diff
@@ -111,6 +111,17 @@ Get the latest ReachView image version and unzip it:
 
 {%endif%}
 
+!!! note ""
+	Reach Firmware Flash Tool creates 2 folders: one to download the image and another one to unpack it. Once you close the Flasher tool, the folder with the unpacked image will be deleted.
+
+	??? note "Folder Paths for Windows"
+		Downloads folder: `C:\Users\<username>\Downloads\Reach Firmware Flash Tool`  
+		Unpacking folder: `C:\Users\<username>\AppData\Local\Temp\Reach Firmware Flash Tool`
+
+	??? note "Folder Paths for Linux"
+		Downloads folder: `/home/<username>/Downloads/Reach Firmware Flash Tool`  
+		Unpacking folder: `/tmp/Reach Firmware Flash Tool`
+
```

Comment:
I suggest putting this notion after `Download the proposed ReachView firmware` screenshot.
@TatianaAndreeva what do you think?

Reply:
I agree, this sounds reasonable